iot_config/src/helium_netids.rs

```diff
@@ -0,0 +1,377 @@
+use crate::lora_field::{self, DevAddrConstraint, LoraField, NetIdField};
+use helium_proto::services::iot_config::org_create_helium_req_v1::HeliumNetId as ProtoNetId;
+use std::{collections::HashSet, ops::RangeInclusive};
+
+const TYPE_0_ID: NetIdField = LoraField(0x00003c);
```

Comment:
I'm not sure how to properly identify it, but we still have to support `000024` as a NetID as well. At least until everyone can be migrated away to `00003c`.

Reply:
i assumed that if these are the three official helium netids then these would be the ones we would need to make extensible and manage ongoing; i don't believe there are any changes in this PR that _don't_ support `000024` in the same fashion they've already been supported: orgs can continue to assign devaddrs within slices of that address space, they just can't extend within that space. once everyone is migrated they're just database entries we can keep or delete as needed